mmocr/utils/ocr.py

```diff
@@ -400,6 +406,38 @@ def __init__(self,
             if hasattr(model, 'module'):
                 model = model.module
 
+    @staticmethod
+    def tesseract_det_inference(imgs):
+        """Inference image(s) with the tesseract detector
+
+        Args:
+            imgs (str/ndarray or list[str/ndarray] or tuple[str/ndarray]):
+                Either image files or loaded images.
+        Returns:
```

Comment:
```suggestion

        Returns:
```
Need a blank line here